parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -201,6 +201,23 @@ installImg() {
     retrycmd_get_executable 120 5 $img_filepath "https://acs-mirror.azureedge.net/img/img-linux-amd64-v0.5.6" ls || exit $ERR_IMG_DOWNLOAD_TIMEOUT
 }
 
+extractKubeBinaries() {
+    # If the version has a trailing .1 remove it,
+    # acs-mirror.azureedge.net does not have the base image patch .1 nomenclature at the end
+    if (($(echo ${KUBERNETES_VERSION} | tr -d -c "." | wc -m) > 2 && $(echo ${KUBERNETES_VERSION} | rev | cut -d"." -f 1) == 1)); then
+      KUBERNETES_VERSION=$(echo ${KUBERNETES_VERSION} | rev | cut -d"." -f 2- | rev)
```

Comment:
I think we should let the caller pass in a valid kuberntes_version so we don't process it here.

FYI - the k8s version in CSE is always in valid format, 1.17.4, it's only the VHD baking code part (which should be processed before passing to this method)